stl/inc/forward_list

```diff
@@ -1491,6 +1495,18 @@ template <class _Iter, class _Alloc = allocator<_Iter_value_t<_Iter>>,
 forward_list(_Iter, _Iter, _Alloc = _Alloc())->forward_list<_Iter_value_t<_Iter>, _Alloc>;
 #endif // _HAS_CXX17
 
+#if _HAS_CXX20
+template <class _Ty, class _Alloc, class _Pr>
+void erase_if(forward_list<_Ty, _Alloc>& _Cont, _Pr _Pred) {
+    _Cont.remove_if(_Pass_fn(_Pred));
+}
+
+template <class _Ty, class _Alloc, class _Uty>
+void erase(forward_list<_Ty, _Alloc>& _Cont, const _Uty& _Val) {
+    _Cont.remove_if([&](_Ty& _Elem) { return _Elem == _Val; });

```

Comment:
Because the Standard says "*Effects:* Equivalent to: `erase_if(c, [&](auto& elem) { return elem == value; });`" and the difference in constness is observable. @SuperWig's code is correct as written and should not be changed.

Reply:
That's a good reason. Is there any reason the working draft shouldn't be changed?